File: P4.c
// this is a program to take two variables from the user and give its result
#include <stdio.h>
void main()
{
    int a,b,c;
    printf("enter the first number");
    scanf("%d",& a);
    printf("enter the second number");
    scanf("%d",& b);
    c = a+b;
    printf("sum of %d and %d is %d",a,b,c);
}

File: P4.java
// this is the program for addition of two numbers
import java.util.*;
public class P4
{
    public static void main(String args[])
    {
        Scanner sc = new Scanner(System.in);
        int a,b,c;
        System.out.println("enter the first number");
        a = sc.nextInt();
        System.out.println("enter the second number");
        b = sc.nextInt();
        c = a+b;
        System.out.println("the sum of "+a+" and "+b+" is "+c);
    }
}